cfme/utils/appliance/__init__.py

```diff
@@ -502,9 +502,11 @@ def configure(self, log_callback=None, **kwargs):
             if fix_ntp_clock and not self.is_pod:
                 self.fix_ntp_clock(log_callback=log_callback)
                 # TODO: Handle external DB setup
-            # This is workaround for Openstack appliances to use only one disk for the VMDB
-            if on_openstack and self.is_downstream and not self.unpartitioned_disks:
-                self.configure_rhos_db_disk()
+            # This is workaround for appliances to use only one disk for the VMDB
+            # If they have been provisioned with a second disk in the infra,
+            # 'self.unpartitioned_disks' should exist and therefore this won't run.
+            if self.is_downstream and not self.unpartitioned_disks:
```

Comment:
Applicable on any appliance, only requirement is the base disk the VM is using needs to have some free unallocated space on it for this to work. So we have 2 options:
1) attach a separate DB disk
2) increase size of the base disk, this will partition it for the DB

If there's no attached DB disk and no free space, an error is logged by our scripts here which states that CFME DB setup is going to fail.